### PR TITLE
Rune scimitar's damage is now 28 from 35

### PR DESCRIPTION
**Pull request by Takahiru**
# General Documentation

### Intent of your Pull Request

Reduces the force of the rune scimitar from 35 to 28.

### Why is this change good for the game?
Nerfs a weapon miners get that is almost as good as a desword.
# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Changes rune scimitar damage to 28 from 35. It's a necropolis chest drop.

### What general grouping does this PR fall under? 
Mining change

# Changelog

:cl:  
tweak: rune scimitar now does 28 damage.  
/:cl:

### CODE REVIEW

**Comment by Hopekz** [comment]
I still don't know why miners even get a sword. They'd never use it for boss fighting, and I've never seen them not use ranged on regular fauna because they don't want to get hit

**Comment by Hopekz** [comment]
I still don't know why miners even get a sword. They'd never use it for boss fighting, and I've never seen them not use ranged on regular fauna because they don't want to get hit

Unless this one is specifically for ashwalkers then it's also weird being in this chest where miner stuff is expected

**Comment by ChesterTheCheesy** [comment]
But rune scimmy! I use it to chop watchers when i don't use crusher!

**Review by SomeguyManperson — APPROVED** [review]
